<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiceGoblin</class>
 <widget class="QMainWindow" name="DiceGoblin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dice Goblin : A RichPresence customiser </string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textDetail">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>321</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textStat">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>321</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Stat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonUpdate">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>88</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>button label</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textUrl">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>button url</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
